src/Microsoft.PowerShell.Commands.Utility/project.json

```diff
@@ -105,8 +105,8 @@
             },
             "imports": [ "dnxcore50", "portable-net45+win8" ],
             "dependencies": {
-                "Microsoft.CodeAnalysis.CSharp": "1.3.1-beta1-20160616-03",
-                "System.Diagnostics.TextWriterTraceListener": "4.3.0-preview1-24530-04"
+                "Microsoft.CodeAnalysis.CSharp": "1.3.2",
```

Comment:
Since we are using the latest version on nuget.org for other packages, shall we use the [`2.0.0-rc` version](https://www.nuget.org/packages/Microsoft.CodeAnalysis.CSharp/2.0.0-rc) for `Microsoft.CodeAnalysis.CSharp`? The `2.0.0-rc` was published on Nov 18.